src/detection_utils/boxes.py

```diff
@@ -87,8 +92,8 @@ def generate_targets(
         anchor_boxes: ndarray,
         truth_boxes: ndarray,
         labels: ndarray,
-        pos_thresh: float = 0.3,
-        neg_thresh: float = 0.2,
+        pos_thresh: float = DEFAULT_POS_THRESHOLD,
+        neg_thresh: float = DEFAULT_NEG_THRESHOLD,
```

Comment:
do these resolve into values in the docs pages?

Reply:
Yep, they will